### PR TITLE
Fix more FastRTPS install commands

### PR DESCRIPTION
**Pull request by julianoes**
Follow up to https://github.com/PX4/Devguide/pull/705.

### CODE REVIEW

**Review by TSC21 — CHANGES_REQUESTED** [review]
We have to install and use FastCDR 1.0.8 and FastRTPS 1.6.0.

**Comment by julianoes** [comment]
@TSC21 huh? I didn't change anything in that respect.

**Comment by TSC21** [comment]
Oh sorry I didn't see all changes. Do you mind changing the versions though?

**Review by TSC21 — APPROVED** [review]

**Comment by julianoes** [comment]
No, I would have to test it to change the versions, and I don't have time to for now, sorry.

File: kr/setup/dev_env_linux_ubuntu.md [hunk 1]
```diff
@@ -101,9 +101,8 @@ mv eprosima_fastrtps-1-5-0-linux-tar-gz eprosima_fastrtps-1-5-0-linux.tar.gz
 tar -xzf eprosima_fastrtps-1-5-0-linux.tar.gz eProsima_FastRTPS-1.5.0-Linux/
 tar -xzf eprosima_fastrtps-1-5-0-linux.tar.gz requiredcomponents
 tar -xzf requiredcomponents/eProsima_FastCDR-1.0.7-Linux.tar.gz
-cd eProsima_FastCDR-1.0.7-Linux; ./configure --libdir=/usr/lib; make; sudo make install
-cd ..
-cd eProsima_FastRTPS-1.5.0-Linux; ./configure --libdir=/usr/lib; make; sudo make install
+(cd eProsima_FastCDR-1.0.7-Linux && ./configure --libdir=/usr/lib && make && sudo make install)
```
Comment (hamishwillee) [inline kr/setup/dev_env_linux_ubuntu.md:104]: FYI only, in future don't bother making any changes to translation branches. The round trip to translation service fixes those up for us. If you make the change then it means I have to resolve a conflict :-(.

**Review by hamishwillee — APPROVED** [review]

**Comment by hamishwillee** [comment]
Thanks. Looks good. I merged on assumption that scripts are good, and I'm testing after the fact. Note that all I'm doing is testing that the scripts will set up a ubuntu environment and start gazebo. No idea how you would test FastRTPS.

Edited: Ubuntu script installs and works with Gazebo. Did not specifically test FastRTPS.

**Pull request merged**